enhancements/observability/cluster-observability-operator-logging-stack.md

```diff
@@ -0,0 +1,405 @@
+
+As a result, to enhance user experience with OpenShift Observability products this proposal aims to be a foundation for future observability enhancements on:
+
+* How to declaratively install/upgrade each Observability product without applying separate procedures each time using COO only.
+* How to uniformly handle the lifecycle, version skew and OpenShift Console dependencies of all Observability view plugins.
+* How to add more value across signals applying automation (i.e. enable correlation components, sharing scheduling primitives, labeling rules/alerts).
+
+Notwithstanding this proposal is limited to accomplish the above for the OpenShift Logging product only.
+
+![coo-vision](./assets/coo-vision.png)
+
+### User Stories
+
+As an OpenShift administrator, I want to install the logging capability creating a single custom resource, so I can easily set up custom-tailored logging for infrastructure and workloads.
+
+As an OpenShift administrator, I want to upgrade the logging capability editing a custom resource, so I can focus only on post-upgrade steps.
+
+As an OpenShift administrator, I want to edit loggging components through underlying custom resoures, so I can improve overall reliability and performance of individual components when needed.
+
+As an OpenShift administrator, I want to use metrics to logs correlation without additional installation procedure, so I can troubleshoot infrastructure/workloads by navigating from metrics to logs and vice versa.
+
+### Goals
+
+* Provide custom resources managed by the COO that are responsible for installing and upgrading the OpenShift Logging stack (i.e. operators and operands).
+* Provide a custom resource managed by the COO that is responsible for installing and upgrading the OpenShift Console logging UI plugin.
+* Provide automation for installing and upgrading correlation components when metrics and logging are available.
+
+### Non-Goals
+
+* Provide opinionated abstractions in form of custom resources to replace the OpenShift Logging custom resources.
+* Provide replacement installations/upgrading procedures for the Operator Framework and Lifecycle Manager (OLM) to manage operators.
+
+## Proposal
+
+Two CRDs will be introduced namely `OperatorConfig` to manage installation of dependent operators (i.e. CLO, LO) and `LoggingStack` to manage installation of the OpenShift Loggging stack itself. In addition the relation between the `LoggingStack` and the `UIPlugin` resources will be outlined on how to manage the [logging-view-plugin](logging-view-plugin) deployment. Finally the automation to manage and the [korrel8r](korrel8r) deployment for metrics-to-logs correlation will be introduced in the COO when a least a `MonitoringStack` and a `LoggingStack` resource are created by a user.
```

Comment:
Do we need to consider calling this "ObservabilityOperatorConfig" ?  I know we can add it into an observability group but do we run the risk of users thinking this generally applies to "operators"?